pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/bestpractices/ApexUnitTestClassShouldHaveAssertsRule.java

```diff
@@ -37,6 +37,24 @@ public class ApexUnitTestClassShouldHaveAssertsRule extends AbstractApexUnitTest
         ASSERT_METHODS.add("system.assert");
         ASSERT_METHODS.add("system.assertequals");
         ASSERT_METHODS.add("system.assertnotequals");
+        ASSERT_METHODS.add("system.assert.areequal");
```

Comment:
This may be a small thing, but there's a comment below this with "Fully-qualified variants...". Should the qualified names of the new methods be moved under that comment for consistency?